worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1974,7 +2065,7 @@ public int fixLiquid(BlockVector3 origin, double radius, BlockType fluid) throws
 
         // There are boundaries that the routine needs to stay in
         Mask mask = new MaskIntersection(
-                new BoundedHeightMask(getWorld().getMinY(), Math.min(origin.getBlockY(), getWorld().getMaxY())),
+                new BoundedHeightMask(getWorld().getMinY(), Math.min(origin.getBlockY(), getWorld().getMaxY())), //TODO Support for new world heights.
```

Comment:
Isn't that covered within dords' pr? If so this todo can be removed.

Reply:
yes